app/src/main/java/org/p2p/wallet/history/ui/history/HistoryPresenter.kt

```diff
@@ -69,30 +64,45 @@ class HistoryPresenter(
     }
 
     override fun loadHistory() {
-        if (transactions.isNotEmpty()) {
```

Comment:
You have broke logic which allows us to show history after tab was switched again